tests/models.py

```diff
@@ -12,7 +12,7 @@
 # Django
 from django.conf import settings
 from django.db import models
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import gettext_lazy as _
```

Comment:
[memo]
Replace to `gettext_lazy`.

https://docs.djangoproject.com/en/4.0/releases/4.0/#features-removed-in-4-0